src/components/QrCodeForm.vue

```diff
@@ -0,0 +1,208 @@
+<script setup lang="ts">
+import { pix } from "../utils/QrCodePix";
+import { ref } from "vue";
+import { debounce } from "@/utils/debounce";
+import CustomButton from "./CustomButton.vue";
+import api from "../services/index";
+
+const props = defineProps({
+  pixTarget: String,
+  tokenValue: Number,
+});
+
+console.log(props.tokenValue);
+console.log(props.pixTarget);
+
+const qrCode = ref<string>("");
+const qrCodePayload = ref<string>("");
+const pixQrCode = pix({
+  pixKey: props.pixTarget ?? "",
+  value: props.tokenValue,
+});
+const pixIsValid = ref<number>(0);
```

Comment:
Why are u controlling this variable as a number, I think that a boolean is more concise, because we don't distinguish between what kind of error occurred, or the pix is valid or not. It's better to make some boolean logics in if

Reply:
Also, rename this variable to isPixValid to be more concise